src/Bicep.LangServer/Handlers/BicepHoverHandler.cs

```diff
@@ -77,8 +77,10 @@ public BicepHoverHandler(ISymbolResolver symbolResolver) : base(CreateRegistrati
 
                     return $"```bicep\nmodule {module.Name}\n```";
 
-                case OutputSymbol output:
+                /* TODO discuss whether we need this?
+                case OutputDeclaration output:
                     return $"```bicep\noutput {output.Name}: {output.Type}\n```";
```

Comment:
Hovers are of limited use, because it won't be possible to reference outputs anywhere else in the document so they will only show up when hovering **on** the output itself. I'll try and fix this just for consistency though.

Then there's the question of whether outputs should show up in the 'symbols' view. With this change, outputs are no longer symbols, but I'm not sure whether VSCode's notion of symbols is necessarily the same thing as the strict classical notion of symbols.

Reply:
Discussed offline, I'm going to leave the symbols alone for now.